cluster/examples/kubernetes/ceph/psp/kube-registry.yaml

```diff
@@ -0,0 +1,129 @@
+# Example showing how to run the kube-registry application with pod security policies in effect.
```

Comment:
Could we reference this file from filesystem.md in the section on the registry sample? Otherwise users might not find it.

Reply:
Yeah. Let me add that.